src/Engine/ProtoCore/DSASM/Executive.cs

```diff
@@ -2692,8 +2707,7 @@ private void ExecuteDebug(int exeblock, int entry, List<Instruction> breakpoints
             }
         }
 
-
-        private void Execute(int exeblock, int entry, Language language = Language.kInvalid)
+        private void ExecuteInternal(int exeblock, int entry, Language language = Language.kInvalid)
```

Comment:
Minor refactor - renaming Execute to ExecuteInternal. This signifies that these calls are only from within the current Executive